src/XIVLauncher.Core/Components/MainPage/MainPage.cs

```diff
@@ -162,7 +163,7 @@ private void ProcessLogin(LoginAction action)
         });
     }
 
-    public async Task<bool> Login(string username, string password, bool isOtp, bool isSteam, bool doingAutoLogin, LoginAction action)
+    public async Task<bool> Login(string username, string password, bool isOtp, bool isSteam, bool doingAutoLogin, bool isFreeTrial, LoginAction action)
```

Comment:
To fix the above issue:
```suggestion
    public async Task<bool> Login(string username, string password, bool isOtp, bool isSteam, bool isFreeTrial, bool doingAutoLogin, LoginAction action)
```